### PR TITLE
data:Tk-brainCodes

### PR DESCRIPTION
**Pull request by Tk-brainCodes**
## Changes proposed

<p>Add a new file data/Tk-brainCodes.json which contains my Linkfree profile links</p>

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<p>This PR contains my bio data for Linkfree, in the directory data/Tk-brainCodes .

### CODE REVIEW

**Review by sital002 — CHANGES_REQUESTED** [review]
Your file should be inside data folder instead of ```Tk-brainCodes .json``` it should be ```data/Tk-brainCodes .json````.
Please make the necessary changes to get your profile merged

File: Tk-brainCodes.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Eboreime ThankGod",
+  "bio": "Open Source | Frontend Developer | React js | Next js | Typescript",
+  "links": [
+    {
+      "github": "Github",
```
Comment (sital002) [inline Tk-brainCodes.json:6]:
```suggestion
```
github isnot a valid property 
I will attach a quick guide link here https://www.biodrop.io/docs/quickstart-json

File: Tk-brainCodes.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      "name": "Github Profile",
+      "url": "https://github.com/Tk-brainCodes",
+      "icon": "FaGithub"
+    },
+    {
+      "portfolio": "Portfolio",
```
Comment (sital002) [inline Tk-brainCodes.json:12]:
```suggestion
```
portfolio isnot a valid property

File: Tk-brainCodes.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      "name": "My Website",
+      "url": "https://eboreime-thankgod.vercel.app/",
+      "icon": "FaLink"
+    },
+    {
+      "linkedin": "Linkedin",
```
Comment (sital002) [inline Tk-brainCodes.json:18]:
```suggestion
```

File: Tk-brainCodes.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      "name": "Linkedin",
+      "url": "https://www.linkedin.com/in/eboreime-thankgod-34864a1b1/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "twitter": "Twitter",
```
Comment (sital002) [inline Tk-brainCodes.json:24]:
```suggestion
```

**Comment by SaraJaoude** [comment]
We look forward to seeing your Profile on BioDrop, but as there is no recent activity from you to make the changes suggested I am going to close this pull request.

We hope you will raise a new pull request to create your Profile and when you do, you might want to take a look at t[hese Docs](https://www.biodrop.io/docs/quickstart-json) which will help with the suggested changes the maintainers have proposed. 

Please remember you can also create your Profile using [Forms](https://www.biodrop.io/docs/quickstart-forms).

**Pull request closed without merging**